src/components/Books.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import AddBook from './AddBook';
+import Book from './Book';
+
+const Books = () => (
+  <>
+    <Book title="Book title" author="Book author" />
+    <button type="button">remove</button>
+    <AddBook />
+  </>
```

Comment:
[OPTIONAL]
- [ ] Instead of a single book, Kindly you may consider rendering an array of books as a placeholder.